src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -60,6 +62,10 @@ function ExecutiveEducation2UPage() {
   };
 
   const handleCheckoutSuccess = (response) => {
+    sendEnterpriseTrackEvent(
```

Comment:
Actually, related, this function isn't really about form submission; it's more about what to do _after_ form submission (the redirect). Should this `submitted` event be in `UserEnrollmentForm`'s `handleFormSubmit` function?